website/docs/how-to/how-to-clone-environment.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: How to clone an environment
+---
+
+:::info availability
+
+Environment cloning is planned to be released in **Unleash 4.18**.
+
+:::
+
+Environment cloning is a tool to enable an admin to clone an environment, including all feature toggles and strategies, into a new environment. See [_Cloning environments_](../user_guide/environments#cloning-environments) for more information.
```

Comment:
- It's exactly the same thing but when I read "cloning is a tool to clone" it feels a bit redundant
- We're not technically cloning toggles because you can't do that, only their enabled state, I think not being clear about that has the potential to be very confusing

```suggestion
Environment cloning is a tool to enable an admin to duplicate an existing environment, including all feature toggles strategies and their state. See [_Cloning environments_](../user_guide/environments#cloning-environments) for more information.
```

Reply:
Thanks, much clearer 👍